libs/utils/analysis_module.py

```diff
@@ -41,4 +44,7 @@ def __init__(self, trace):
 
         trace._registerDataFrameGetters(self)
 
+        logging.config.fileConfig('logging.conf')
```

Comment:
Can we avoid calling this more than once? It would be good to be able to edit it easily and have everything truly configured from only one place. If not, could we set the file path with a global variable that we import where we need it? That would make it easier to change. I'm also concerned about using a relative path. If we have the logging config file path as a global variable then we can easily have it set relative to `__file__` or whatever.